packages/api/test/post-build-test.js

```diff
@@ -0,0 +1,115 @@
+
+const { describe, before, after, it } = require('mocha');
+const { Firestore } = require('@google-cloud/firestore');
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+describe('flaky express server', () => {
+  let server;
+  let client;
+  before(async () => {
+    const serverModule = require('../server');
+    server = serverModule.server;
+    client = new Firestore();
+    global.headCollection = 'repositories-testsuite-' + Math.random().toString(36).substr(2, 9);
```

Comment:
I'd pull in the [uuid](https://www.npmjs.com/package/uuid) dependency, and use this to generate a UUID, rather than `Math.random()`.

If you like math, [here's a good read on why](https://medium.com/@betable/tifu-by-using-math-random-f1c308c4fd9d) 😆